site/gatsby-site/src/components/cite/TsneVisualization.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from 'react';
+import { Spinner } from 'react-bootstrap';
+import styled from 'styled-components';
+import { useApolloClient, gql } from '@apollo/client';
+import { Image } from '../../utils/cloudinary';
+import { TransformWrapper, TransformComponent } from 'react-zoom-pan-pinch';
+
+const incidentQuery = gql`
```

Comment:
This was running surprisingly slowly. Maybe time to upgrade our database server instance?

Reply:
Interesting, I did not notice that.